### PR TITLE
improve Symfony 4 compatibility

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -32,11 +32,11 @@
         "twig/twig": "~1.12|~2.0",
         "nelmio/alice": "~1.7|~2.0",
         "jackalope/jackalope-doctrine-dbal": "1.1.*|1.2.*",
-        "doctrine/phpcr-odm": "~1.3",
-        "doctrine/phpcr-bundle": "~1.3",
+        "doctrine/phpcr-odm": "^1.3|^2.0",
```
[the remark this answers is not in the document]
Comment (lsmith77) [reply composer.json]: actually I was meaning to also add `"minimum-stability": "dev"`

**Comment by lsmith77** [comment]
looks like this will also require some work to get https://github.com/brianium/paratest/pull/289 passing :(

**Comment by lsmith77** [comment]
good news https://github.com/brianium/paratest/pull/290

**Comment by Jean85** [comment]
This should be superseeded by #359... I don't think it's wise to support Symfony 4 under the 1.x branch here, too much stuff is not compatible, and a lot of package constraints require a bump.

**Comment by lsmith77** [comment]
agreed

**Pull request closed without merging**